pkg/lib/authn/identity/loginid/errors.go

```diff
@@ -4,5 +4,4 @@ import (
 	"errors"
 )
 
-var ErrValidate = errors.New("ValidateError")
 var ErrNormalize = errors.New("NormalizationError")
```

Comment:
Please also remove this newly introduced error. We no longer need it.

Reply:
Removed `ErrNormalize` related code